LandSandBoat/server
# [lua/sql] Sarcopsylla NM

**_I affirm:_**

## What does this pull request do?

Implements retail accuracy for [Sarcopsylla](https://www.bg-wiki.com/ffxi/Sarcopsylla)

## Steps to test these changes

Goto the mob in grauberg, fight it and see that `!getstats` shows proper data, casts the right set of spells, and it doesn't instagib from JA/crit/ws
